### PR TITLE
Add Piwik statistics to the API, refs etalab/udata#60

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: udata_piwik/client.py [hunk 1]
```diff
@@ -11,31 +11,33 @@
...
-        r = requests.post(self.base_url, params=self.params(method, **kwargs))
-        return r.json()
+def analyze(method, **kwargs):
+    """Retrieve JSON stats from Piwik for a given `method` and parameters."""
+    base_url = 'http://{0}/index.php'.format(current_app.config['PIWIK_URL'])
+    data = {
+        'module': 'API',
+        'idSite': current_app.config['PIWIK_ID'],
+        'method': method,
+        'format': kwargs.pop('format', 'json'),
```
Comment (noirbizarre) [inline udata_piwik/client.py]: No need for token auth ?

File: udata_piwik/tasks.py [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
-from udata.tasks import job
 
+from flask import current_app
+
+from udata.tasks import connect, job
+from udata.api.signals import on_api_call
+from udata.core.dataset.signals import on_dataset_published
+from udata.core.reuse.signals import on_reuse_published
+
+import udata_piwik
```
Comment (noirbizarre) [inline udata_piwik/tasks.py]: why not from udata_piwik import client ?

File: udata_piwik/client.py (hunk 1 above)
Comment (davidbgk) [inline udata_piwik/client.py]: We never send a `token_auth` in `kwargs` so it's a bit useless IMHO, I can keep it though. Thoughts?

Comment (noirbizarre) [inline udata_piwik/client.py]: If it's working without, it's good to me.

**Pull request merged**